lib/utils/computed-property.js

```diff
@@ -29,7 +32,6 @@ function isCPDesc(node, method) {
   return false; // don't know how you could get here
 }
 
-
 function isPrototypeExtCP(node) {
```

Comment:
So by default in Ember `prototype` extensions are turned on, so there is a CP syntax that is like:

```
foo: function() {
  return this.get('bar') * 2;
}.property('bar')
```